### PR TITLE
Fixed JsFiddle links

### PR DESCRIPTION
**Pull request by fazouane-marouane**
There was an error in JsFiddle links handling. Valid links like these were not accepted:
http://jsfiddle.net/Sandhose/BcKhe/1/
http://jsfiddle.net/zgjhjv9j/

With this PR, we can handle pretty much any kind of jsfiddle links (anonymous, revised, etc.)

---

Il y avait une erreur dans le traitement des liens JsFiddle. Plusieurs liens valides n'étaient pas reconnus.
Plus de détails: https://zestedesavoir.com/forums/sujet/1714/ajout-de-html-et-js-dans-le-contenu/?page=5#p45915

### CODE REVIEW

**Comment by cgabard** [comment]
One think must be checked : only readonly fiddle should be allowed to avoid change the content without validation

**Comment by fazouane-marouane** [comment]
Hi
Jsfiddles are versioned. So each time you edit/update a fiddle a new version is created along with a brand new uri. Meaning that jsfiddle uris are already readonly

File: markdown/extensions/video.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def extendMarkdown(self, md, md_globals):
             r'https?://youtu\.be/(?P<youtubeid>\S[^?&/]+)?')
         if self.config["jsfiddle"][0]:
             self.add_inline(md, 'jsfiddle', JsFiddle,
-                r'https?://(www.|)jsfiddle\.net/(?P<jsfiddleid>[a-z0-9]+)/(?P<jsfiddlerev>[0-9]+)/?')
+                r'https?://(www.|)jsfiddle\.net(/(?P<jsfiddleuser>\w+))?/(?P<jsfiddleid>\w+)(/(?P<jsfiddlerev>[0-9]+)|)/?')
```
Comment (cgabard) [inline markdown/extensions/video.py:55]: Why there are a trailling `|` in the end of the re ?

**Comment by cgabard** [comment]
I will refactorize the "video" extension this week and add your modification with additionnals tests. We need to ensure url like "http://jsfiddle.net/<user>/<token>/" without revision are not allowed (because this url can be changed into jsfiddle)

**Comment by cgabard** [comment]
I have included this modification with tests into #86, thank for the PR @fazouane-marouane !

**Pull request closed without merging**